ca_description/urdf/sensors/color_sensor.xacro

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find ca_description)/urdf/common_properties.xacro"/>
+
+  <xacro:macro name="color_sensor" params="prefix parent visualize:=false *origin1 **origin2">  
+    <xacro:property name="parent_link" value="${parent}"/>
+    <xacro:property name="color_sensor_len" value="0.05"/>
```

Comment:
Since you're already inside the color sensor macro, would be better to call this variable `length`?

Reply:
Actually it was used for the visual, but now that we use a `mesh`, it's not needed anymore, deleted.